pkg/utils/k8s_utils.go

```diff
@@ -215,9 +215,9 @@ type DriverConfig struct {
 
 // ReadTestDriverConfig will read the driver config
 func ReadTestDriverConfig(driverconfig string) string {
-	file, err := os.ReadFile(filepath.Clean(driverconfig))
+	file, _ := os.ReadFile(filepath.Clean(driverconfig))
 	var DriverConfig DriverConfig
-	err = yaml.Unmarshal(file, &DriverConfig)
+	err := yaml.Unmarshal(file, &DriverConfig)
```

Comment:
Rest of the changes looks good to me.
Just a small concern here, as we are ignoring the err arg above while trying to read the file for any chance unmarshalling can result into panic if file == nil?
As long as yaml.Unmarshal return error in case when file is nil, we are good.
But as a good practice we should ASAP so may be we can have a check on os.ReadFile as well, what do you think?

Reply:
as this function being used at only one place, i.e. https://github.com/dell/cert-csi/blob/5494e07725e3aefef15c15092ee2d1b8e84b6162/pkg/utils/k8s_utils_test.go#L241, and the path being defined in argument, we can be confident that it won't panic.